src/pointcloud_to_laserscan_node.cpp

```diff
@@ -225,9 +243,80 @@ void PointCloudToLaserScanNode::cloudCallback(
     int index = (angle - scan_msg->angle_min) / scan_msg->angle_increment;
     if (range < scan_msg->ranges[index]) {
       scan_msg->ranges[index] = range;
+      v_pointcloud_index.at(index) = pointcloud_index;
     }
   }
-  pub_->publish(std::move(scan_msg));
+
+  laserscan_pub_->publish(std::move(scan_msg));
+
+  pcl::PointCloud<pcl::PointXYZ> pcl_pointcloud;
+  sensor_msgs::PointCloud2ConstIterator<float> iter_x(*cloud_msg, "x"), iter_y(*cloud_msg, "y"),
```

Comment:
Declaring multiple variables in the same line isn't allowed.
Please see A7-1-7 in https://www.autosar.org/fileadmin/user_upload/standards/adaptive/17-03/AUTOSAR_RS_CPP14Guidelines.pdf.

Reply:
Fixed at https://github.com/tier4/pointcloud_to_laserscan/pull/1/commits/f68d96d6d31cfc7fbc4531e14f53740dd8abad06